### ISSUE TITLE
Add CODEOWNERS to repository

### ISSUE THREAD

**Issue by itsdalmo** [TeliaSoneraNorge/telia-terraform-modules#37]
Never done this before, but it looks good and like something we should be using :)

**Comment by antonbabenko** [comment]
We have it in [devops](https://github.com/TeliaSoneraNorge/devops/blob/master/CODEOWNERS). It has some value.

**Comment by itsdalmo** [comment]
Indeed, I was inspired by the devops and terraform-modules repositories :)
